docs/user-manual/framework/supported-platforms.md

```diff
@@ -0,0 +1,32 @@
+## Supported Platforms
+
+A **supported platform** refers to a combination of hardware and operating system where the software is actively tested through continuous integration (CI). A platform is considered **supported** if all CI tests pass consistently, ensuring compatibility and reliability.  
+
+The following tables list currently supported platforms:
+
+| Hardware | OS | Reference Deployment | CI Passes |
+|------|----------|----|-----------|
+| BeagleBone Black | VxWorks 7 | [`fprime-vxworks-reference`](https://github.com/fprime-community/fprime-vxworks-reference) | TODO |
```

Comment:
Is TODO the want for a badge that indicates status of CI? 

This may help: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge

For example: ![example workflow](https://github.com/nasa/fprime/actions/workflows/ext-build-led-blinker.yml/badge.svg) 

We may want to confirm if this translates well to the website (i.e. outside of GitHub MD rendering).

Reply:
Yes, that is what I was thinking.

Reply:
We should scrub TBDS and TODOS.